feat: migration guide analytics (#830)
ZupIT/beagle-docs: content/en/article/_index.md

---
title: "Articles"
weight: 101
description: >-

>      In this section we publish articles about the Beagle and its features, migration guides for outdated components, etc.

Reviewer: ```suggestion
     In this section we publish articles about the Beagle framework like features, migration guides for outdated functionalities and more.
```
